sharqiyya-app-homepage/style/blocks/dates-bar.scss

```diff
@@ -0,0 +1,79 @@
+.dates-bar {
+  background-color: #2E2972;
+  @include for_phone_only {
+    overflow-x: scroll;
+    overflow-y: hidden;
+  }
+}
+
+.dates-bar__max {
+  display: flex;
+  height: 50px;
+  border-top: 1px solid rgba(0,0,0,0.4);
+  font-family: Montserrat;
+  @include maxwidth
+}
+
+.dates-bar__left {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 25px;
+  flex: 3;
+}
+
+.dates-bar__right {
+  display: flex;
+  align-items: center;
+  padding-left: 20px;
+  background-color: #100D46;
+  flex: 1;
```

Comment:
We shouldn't rely on `flex: 1` for the sizing of the search bar, because we want that to be a fixed width. When resizing it's becoming very small and the text is overflowing